coreos-installer

```diff
@@ -574,8 +575,13 @@ get_ignition_url() {
 #########################################################
 mount_tmpfs() {
     log "Mounting tmpfs"
+    if [ -n "$1" ]; then
+        options="remount,size=${1}m"
```

Comment:
Why the `remount`?

Reply:
If 1 argument is provided, assuming it's new size for `/mnt/dl`, `remount` is required to resize to new size.

Reply:
Resolving.